python/astshim/test.py

```diff
@@ -131,3 +132,66 @@ def checkBasicSimplify(self, amap):
             self.assertEqual(mb.getNout(), cmp3.getNout())
             self.assertEqual(cmp3.getNin(), cmp3simp.getNin())
             self.assertEqual(cmp3.getNout(), cmp3simp.getNout())
+
+
+def makeCoeffs(nIn, nOut):
+    """Make an array of coefficients for astshim.PolyMap for the following equation:
+
+    fj(x) = C0j x0^2 + C1j x1^2 + C2j x2^2...
+    where:
+    * i ranges from 0 to nIn-1
```

Comment:
I would update the comments to match the code, unless you have a strong preference for the code to change.